### PR TITLE
[Notify] JWT's protocol changes

### PR DESCRIPTION
**Pull request by flypaper0**
# Description

Resolves # (issue)

## Due Dilligence

* [ ] Breaking change
* [ ] Requires a documentation update

### CODE REVIEW

File: Sources/WalletConnectSync/Stores/SyncStore.swift [hunk 1]
```diff
@@ -108,6 +108,8 @@ private extension SyncStore {
 
             guard record.store == name else { return }
 
+            print("Received sync object: \(update)")
```
Comment (llbartekll) [inline Sources/WalletConnectSync/Stores/SyncStore.swift:111]: add logger?

**Review by llbartekll — APPROVED** [review]

File: Sources/WalletConnectNotify/Client/Common/DeleteNotifySubscriptionSubscriber.swift [hunk 2]
```diff
@@ -23,13 +23,21 @@ class DeleteNotifySubscriptionSubscriber {
...
+
                 logger.debug("Peer deleted subscription")
+
+                guard let (_, _) = try? NotifyDeleteResponsePayload.decodeAndVerify(from: payload.request)
+                else { fatalError() /* TODO: Handle error */ }
+
                 let topic = payload.topic
                 networkingInteractor.unsubscribe(topic: topic)
+
+                // TODO: Why two deletes?
```
[the remark this answers is not in the document]
Comment (flypaper0) [reply Sources/WalletConnectNotify/Client/Common/DeleteNotifySubscriptionSubscriber.swift]: removed

File: Sources/WalletConnectNotify/Client/Wallet/ProtocolEngine/wc_notifyMessage/NotifyMessageSubscriber.swift [hunk 3]
```diff
@@ -23,17 +29,43 @@ class NotifyMessageSubscriber {
...
+                            response: response,
+                            protocolMethod: NotifyMessageProtocolMethod()
+                        )
 
-                let record = NotifyMessageRecord(id: payload.id.string, topic: payload.topic, message: messagePayload.message, publishedAt: payload.publishedAt)
-                notifyStorage.setMessage(record)
-                notifyMessagePublisherSubject.send(record)
+                        logger.debug("Sent Notify Receipt Response")
+                    } catch {
+                        fatalError() /* TODO: Handle error */
```
[the remark this answers is not in the document]
Comment (flypaper0) [reply Sources/WalletConnectNotify/Client/Wallet/ProtocolEngine/wc_notifyMessage/NotifyMessageSubscriber.swift:66]: Used for debug. Removed

File: Sources/WalletConnectSync/Stores/SyncStore.swift (hunk 1 above)
Comment (flypaper0) [reply Sources/WalletConnectSync/Stores/SyncStore.swift:111]: same

**Pull request merged**